public_html/assets/js/fa-icons.json

```diff
@@ -0,0 +1 @@
 fa-rupee-sign": "currency indian inr money", "fas fa-sad-cry": "emoticon face tear tears", "far fa-sad-cry": "emoticon face tear tears", "fas fa-sad-tear": "emoticon face tear tears", "far fa-sad-tear": "emoticon face tear tears", "fab fa-safari": "browser", "fab fa-salesforce": "", "fab fa-sass": "", "fas fa-satellite": "communications hardware orbit space", "fas fa-satellite-dish": "SETI communications hardware receiver saucer signal space", "fas fa-save": "disk download floppy floppy-o", "far fa-save": "disk download floppy floppy-o", "fab fa-schlix": "", "fas fa-school": "building education learn student teacher", "fas fa-screwdriver": "admin fix mechanic repair settings tool", "fab fa-scribd": "", "fas fa-scroll": "Dungeons & Dragons announcement d&d dnd fantasy paper script", "fas fa-sd-card": "image memory photo save", "fas fa-search": "bigger enlarge find magnify preview zoom", "fas fa-search-dollar": "bigger enlarge find magnify money preview zoom", "fas fa-search-location": "bigger enlarge find magnify preview zoom", "fas fa-search-minus": "minify negative smaller zoom zoom out", "fas fa-search-plus": "bigger enlarge magnify positive zoom zoom in", "fab fa-searchengin": "", "fas fa-seedling": "flora grow plant vegan", "fab fa-sellcast": "eercast", "fab fa-sellsy": "", "fas fa-server": "computer cpu database hardware network", "fab fa-servicestack": "", "fas fa-shapes": "blocks build circle square triangle", "fas fa-share": "forward save send social", "fas fa-share-alt": "forward save send social", "fas fa-share-alt-square": "forward save send social", "fas fa-share-square": "forward save send social", "far fa-share-square": "forward save send social", "fas fa-shekel-sign": "currency ils money", "fas fa-shield-alt": "achievement award block defend security winner", "fas fa-ship": "boat sea water", "fas fa-shipping-fast": "express fedex mail overnight package ups", "fab fa-shirtsinbulk": "", "fas fa-shoe-prints": "feet footprints steps walk", "fab fa-shopify": "", "fas fa-shopping-bag": "buy checkout grocery payment purchase", "fas fa-shopping-basket": "buy checkout grocery payment purchase", "fas fa-shopping-cart": "buy checkout grocery payment purchase", "fab fa-shopware": "", "fas fa-shower": "bath clean faucet water", "fas fa-shuttle-van": "airport machine public-transportation transportation travel vehicle", "fas fa-sign": "directions real estate signage wayfinding", "fas fa-sign-in-alt": "arrow enter join log in login sign in sign up sign-in signin signup", "fas fa-sign-language": "Translate asl deaf hands", "fas fa-sign-out-alt": "arrow exit leave log out logout sign-out", "fas fa-signal": "bars graph online reception status", "fas fa-signature": "John Hancock cursive name writing", "fas fa-sim-card": "hard drive hardware portable storage technology tiny", "fab fa-simplybuilt": "", "fab fa-sistrix": "", "fas fa-sitemap": "directory hierarchy ia information architecture organization", "fab fa-sith": "", "fas fa-skating": "activity 
```

Comment:
for entries without a search term (e.g. 500px), the value should be the second part of the key without the `fa-`. Otherwise they will not come up in the search, I think.

Reply:
Actually, we should add that, to all entries, otherwise you will not find exact matches on the name,. e.g. "x-ray" (if I understand the search function correctly)

Reply:
Not needed, as I made the search box look both in the search terms but also the class names: `.popover-body div button[data-classname*='"+search_term+"']`

https://github.com/nf-core/nf-co.re/pull/334/files#diff-50fe998361a51c27bb42f5df30467eeaR90